### PR TITLE
4038 url changes

### PR DESCRIPTION
**Pull request by kialj876**
*Issue #:* /bcgov/entity#4038

*Description of changes:*
- updated k8s for missing params and url change
- updated local config for url change

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by severinbeauvais** [comment]
I added Cam as a reviewer since we were talking about some of stuff yesterday.

File: k8s/templates/business-create-ui.dc-srvc-rt-cmap-kcmap.json [hunk 1]
```diff
@@ -276,63 +276,63 @@
...
             "value": ""
         },
         {
-            "name": "REPORT_SVC_URL",
-            "displayName": "Report Service API URL",
-            "description": "The URL to the Report Service API.",
-            "required": false,
+            "name": "SBC_CONFIG_AUTH_API_URL",
+            "displayName": "Auth URL",
+            "description": "The uth URL with no slash.",
```
Comment (ochiu) [inline k8s/templates/business-create-ui.dc-srvc-rt-cmap-kcmap.json:334]: Just a typo in the description. 

Also, I thought I had already made this change in another ticket but I must have accidentally reset it locally and it didn't make it into my PR. So thanks for catching that!

File: k8s/business-create-ui.dc-srvc-rt-cmap-kcmap.param [hunk 2]
```diff
@@ -4,32 +4,36 @@
 # Component: deployment config, service, route, config map, kc config map
 # Action: build
 # Template File: templates/business-create-ui.dc-srvc-rt-cmap-kcmap.json
+# Hint: oc process -n mpgxod-dev -f templates/business-create-ui.dc-srvc-rt-cmap-kcmap.json --param-file=business-create-ui.dc-srvc-rt-cmap-kcmap.param
```
Comment (severinbeauvais) [inline k8s/business-create-ui.dc-srvc-rt-cmap-kcmap.param:7]: Nice! 👍

File: k8s/templates/business-create-ui.dc-srvc-rt-cmap-kcmap.json (hunk 1 above)
Comment (severinbeauvais) [inline k8s/templates/business-create-ui.dc-srvc-rt-cmap-kcmap.json:334]: uth -> Auth

File: k8s/business-create-ui.dc-srvc-rt-cmap-kcmap.param [hunk 3]
```diff
@@ -4,32 +4,36 @@
...
-DASHBOARD_URL="https://test.bcregistry.ca/cooperatives/"
-KEYCLOAK_CONFIG_PATH="https://test.bcregistry.ca/cooperatives/auth/config/kc/keycloak.json"
-ADDRESS_COMPLETE_KEY=""
-LD_CLIENT_ID=""
-SENTRY_DSN=""
-REALM="fcf0kpqr"
-AUTH_SERVER_URL="https://sso-test.pathfinder.gov.bc.ca/auth"
+BUSINESSES_URL="https://dev.bcregistry.ca/businesses/auth/"
+AUTH_URL="https://dev.bcregistry.ca/businesses/auth/"
+DASHBOARD_URL="https://dev.bcregistry.ca/businesses/"
```
Comment (severinbeauvais) [inline k8s/business-create-ui.dc-srvc-rt-cmap-kcmap.param:25]: I think these 3 URLs should be /business/ (not plural).
Also line 30 below.

File: k8s/business-create-ui.dc-srvc-rt-cmap-kcmap.param [hunk 4]
```diff
@@ -4,32 +4,36 @@
...
 # Action: build
 # Template File: templates/business-create-ui.dc-srvc-rt-cmap-kcmap.json
+# Hint: oc process -n mpgxod-dev -f templates/business-create-ui.dc-srvc-rt-cmap-kcmap.json --param-file=business-create-ui.dc-srvc-rt-cmap-kcmap.param
+# Hint: oc process -n mpgxod-dev -f templates/business-create-ui.dc-srvc-rt-cmap-kcmap.json --param-file=business-create-ui.dc-srvc-rt-cmap-kcmap.param | oc replace -n mpgxod-dev -f -
 #=========================================================
 NAME="business-create"
 DOCKER_REGISTRY="docker-registry.default.svc:5000"
 IMAGE_NAMESPACE="mpgxod-tools"
-TAG_NAME="test"
+TAG_NAME="dev"
```
Comment (severinbeauvais) [inline k8s/business-create-ui.dc-srvc-rt-cmap-kcmap.param:13]: I didn't find this param file for Test or Prod. Are they needed?

File: k8s/business-create-ui.dc-srvc-rt-cmap-kcmap.param (hunk 3 above)
Comment (kialj876) [reply k8s/business-create-ui.dc-srvc-rt-cmap-kcmap.param:25]: Oh shoot -> will update!

File: k8s/business-create-ui.dc-srvc-rt-cmap-kcmap.param (hunk 4 above)
Comment (kialj876) [reply k8s/business-create-ui.dc-srvc-rt-cmap-kcmap.param:13]: no we just need the one param file (at least that's how we're doing it in the backend). Less chance of making mistakes across configs that way and you have to fill in the secret values anyways

**Comment by kialj876** [comment]
> Some comments.
> 
> Also, the URL for this UI isn't changing, right? I'm just wondering about `src/vue.config.js` ...?

No this one doesn't change (according to my ticket). The filings ui does though

File: k8s/business-create-ui.dc-srvc-rt-cmap-kcmap.param (hunk 4 above)
Comment (severinbeauvais) [reply k8s/business-create-ui.dc-srvc-rt-cmap-kcmap.param:13]: What about all the environment-specific values specified below in this param file?

Comment (kialj876) [reply k8s/business-create-ui.dc-srvc-rt-cmap-kcmap.param:13]: cmd f 'dev' replace with 'test'/'prod' when you need is safer than having 3 param files that you need to keep in sync I think (all the other env specific ones are secrets). Already people forget to update just the 1 file. If even 1 out of 3 were out of sync it'd be a huge headache for ops on releases

**Pull request merged**